spec/lib/reservation_creator_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe ReservationCreator do
+  let!(:current_user) { UserMock.new }
+  describe 'create!' do
+    shared_examples 'successful create' do |cart:, **attrs|
+      it do
+        creator = ReservationCreator.new(cart: instance_spy('Cart', **cart),
+                                         current_user: current_user,
+                                         **attrs)
+        results = creator.create!
+        expect(results[:result]).to be_truthy
```

Comment:
Wait, I feel like this discussion wasn't resolved. I think was asking if we should also be asserting against the actual value of `results[:results]`, not just whether or not it's truthy. Am I missing something?